requirements.txt

```diff
@@ -3,4 +3,4 @@ python-telegram-bot
 opencv-python
 numpy
 Image
```

Comment:
these 3 requirements can remove

Reply:
sorry, only `opencv-python` and `numpy`, since we still need `Image` for the resizing. 

as long as the filenames and stuff in the temp folder is maintained, this looks good to go

Reply:
Do we need to add a img/ file to the directory? Didn't see any mkdir commands and I had to add one myself